sockeye/arguments.py

```diff
@@ -369,6 +369,11 @@ def add_training_args(params):
                               default=13,
                               help='Random seed. Default: %(default)s.')
 
+    train_params.add_argument('--keep-last-params',
+                              type=int,
+                              default=20,
```

Comment:
looking at an example model that trained for 188 checkpoints and with an average strategy of 'best' with 8 checkpoints, the earliest checkpoint we use for averaging is 168. So 20 would still be fine. That said, I'd still increase the default to 30-40 to be on the save side.